### PR TITLE
Reverse replication documentation updates and merge to new doc structure

### PR DESCRIPTION
**Pull request by aksharauke**
Merges the reverse replication documents to the main doc structure
Limitation section is moved from the runner document to user guide
User guide is having more info on tracking progress at each step
The troubleshooting scenarios are made more defined
Better images added to aid user to track metrics

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ It can also be used to do small scale on-prem POC migrations to get a feel of Sp
 
 - **POC Migration** -  This mode is useful to get up and running quickly to get a feel of what migrating to Spanner would look like. This mode of migration uses the local machine's resources (on which SMT is running) to write data to Spanner. This is an offline migration for migrating moderate-size datasets to Spanner(up to about 100GB). More details about POC migrations are [here](./poc/poc.md).
 
+### Reverse Replication
+
+Spanner migration tool will soon support reverse replication via the UI. Currently it support it only via CLI. More details about reverse replication are [here](./reverse-replication/ReverseReplication.md).
```
Comment (bharadwaj-aditya) [inline docs/index.md:42]: Let's not comment on what is soon to come. I think it makes sense to just say that -
To launch the reverse replication, use the CLI (details link)

File: docs/reverse-replication/ReverseReplicationUserGuide.md [hunk 2]
```diff
@@ -40,31 +56,45 @@ A few prerequisites must be considered before starting with reverse replication.
...
+#### Pub/Sub metrics
+
+Track the Pub/Sub topic [metrics](https://cloud.google.com/pubsub/docs/monitor-topic) to verify that there is inflow of messages by checking the 'Published Messages' metric.
+
+Note that subscription [metrics](https://cloud.google.com/pubsub/docs/monitor-subscription) can be verified to check if the dataflow job that writes to source database is reading from Pub/Sub as expected.
+
 #### Metrics for Dataflow job that writes to source database
 
-The Dataflow job that writes to source database exposes per shard metric like so:
+The Dataflow job that writes to source database exposes per shard metric like so, which should be visible on the right pane titled 'Job Info'.
```
Comment (bharadwaj-aditya) [inline docs/reverse-replication/ReverseReplicationUserGuide.md:97]: Please move the images out of the repo as it adds to bloat on the repo.

**Review by bharadwaj-aditya — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (aksharauke) [reply docs/index.md]: done

File: docs/reverse-replication/ReverseReplicationUserGuide.md [hunk 3]
```diff
@@ -92,31 +122,53 @@ mean(value.user_counter)]
...
 
-     If the forward migration is still running post cutover, the incoming writes on Spanner that are reverse-replicated to the source get forward migrated again. This can cause the load on Spanner to be almost double the expected QPS, as each write will get reflected twice. Also, it could lead to transient inconsistencies in data under certain cases. To avoid this, stop/delete the forward migration pipeline post cutover. If the forward pipeline is required, add custom filtration rules and build a custom forward migration dataflow template.
+#### There is higher load than the expected QPS on  spanner instance post cutover
+
+1. If the forward migration is still running post cutover, the incoming writes on Spanner that are reverse-replicated to the source get forward migrated again. This can cause the load on Spanner to be almost double the expected QPS, as each write will get reflected twice. Also, it could lead to transient inconsistencies in data under certain cases. To avoid this, stop/delete the forward migration pipeline post cutover. If the forward pipeline is required, add custom filtration rules and build a custom forward migration dataflow template.
```
[the remark this answers is not in the document]
Comment (aksharauke) [reply docs/reverse-replication/ReverseReplicationUserGuide.md]: Done

File: docs/reverse-replication/ReverseReplicationUserGuide.md (hunk 2 above)
Comment (aksharauke) [reply docs/reverse-replication/ReverseReplicationUserGuide.md:97]: Will move them once the images get approved

Comment (aksharauke) [reply docs/reverse-replication/ReverseReplicationUserGuide.md:97]: Done, have removed the local images

**Review by bharadwaj-aditya — APPROVED** [review]
LGTM

**Pull request merged**